homeassistant/components/light/greenwave.py

```diff
@@ -65,9 +65,9 @@ def __init__(self, light, host, token, gatewaydata):
         self._name = light['name']
         self._state = int(light['state'])
         self._brightness = greenwave.hass_brightness(light)
-        self._host = host
+        self._host = str(host)
         self._online = greenwave.check_online(light)
-        self._token = token
+        self._token = str(token)
```

Comment:
This should be fixed upstream. 

https://github.com/dfiel/greenwavereality/issues/2